src/inference/src/threading/ie_istreams_executor.cpp

```diff
@@ -15,16 +15,20 @@
 #include "ie_parameter.hpp"
 #include "ie_plugin_config.hpp"
 #include "ie_system_conf.h"
+#include "openvino/runtime/properties.hpp"
 
 namespace InferenceEngine {
 IStreamsExecutor::~IStreamsExecutor() {}
 
-std::vector<std::string> IStreamsExecutor::Config::SupportedKeys() {
+std::vector<std::string> IStreamsExecutor::Config::SupportedKeys() const {
     return {
         CONFIG_KEY(CPU_THROUGHPUT_STREAMS),
         CONFIG_KEY(CPU_BIND_THREAD),
         CONFIG_KEY(CPU_THREADS_NUM),
         CONFIG_KEY_INTERNAL(CPU_THREADS_PER_STREAM),
+        ov::streams::num.name(),
```

Comment:
can we somehow auto-matically concatenated rather than explicitly list the streams' settings, smth like this https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/auto/plugin.cpp#L57

Reply:
otherwise when we add/remove settings to the streams executor we may want to exlicitly update such places

Reply:
This PR is just an adjustment with the new API. Has other PR with new internal properties binding proposal https://github.com/openvinotoolkit/openvino/pull/9679